domain-model/src/test/java/uk/gov/hmcts/cmc/domain/models/statementofmeans/StatementOfMeansTest.java

```diff
@@ -125,8 +125,7 @@ public void shouldBeInvalidForEmptyBankAccount() {
         Set<String> errors = validate(statementOfMeans);
         //then
         assertThat(errors)
-                .hasSize(1)
-                .contains("bankAccounts : may not be empty");
+                .doesNotContain("bankAccounts : may not be empty");
```

Comment:
Please use ` assertThat(response).hasSize(0)` to check there is no error.

Reply:
Ok 
```suggestion
                .hasSize(0);
```